Samples/WinMLSamplesGallery/TransformAsync/SegmentModel.h

```diff
@@ -0,0 +1,152 @@
+#define USE_PIX
+//#define DBG
+#include <winrt/Microsoft.AI.MachineLearning.Experimental.h>
+#include <winrt/Microsoft.AI.MachineLearning.h>
+#include <Windows.AI.MachineLearning.native.h>
+#include <winrt/Windows.Graphics.DirectX.Direct3D11.h>
+#include <windows.media.core.interop.h>
+#include <winrt/Windows.Devices.Display.Core.h>
+#include <strsafe.h>
+#include <wtypes.h>
+#include <winrt/base.h>
+#include <dxgi.h>
+#include <d3d11.h>
+#include <mutex>
+#include <winrt/windows.foundation.collections.h>
+#include <winrt/Windows.Media.h>
+//#include <DXProgrammableCapture.h>
+#include "External/common.h"
+
+
+using namespace winrt::Microsoft::AI::MachineLearning;
+using namespace winrt::Microsoft::AI::MachineLearning::Experimental;
+using namespace winrt::Windows::Graphics::DirectX::Direct3D11;
+using namespace winrt::Windows::Media;
+
+// Model-agnostic helper LearningModels
+LearningModel Normalize0_1ThenZScore(long height, long width, long channels, const std::array<float, 3>& means, const std::array<float, 3>& stddev);
+LearningModel ReshapeFlatBufferToNCHW(long n, long c, long h, long w);
+LearningModel Invert(long n, long c, long h, long w);
+
+class StreamModelBase
+{
+public:
+	StreamModelBase(): 
+		m_inputVideoFrame(NULL),
```

Comment:
>NULL [](http://example.com/codeflow?start=20&length=4)

`nullptr` here too.

Though, it's really more robust to just initialize the field directly rather than at the constructor, which can be brittle in case you add a new field and forget one constructor later.

```
 VideoFrame     m_inputVideoFrame = nullptr;
```

Double though, what type is `VideoFrame`? Is it already a smart pointer that default-initializes itself? If so, we don't need redundant nullification in either place.

Reply:
I'll replace with nullptr and remove uncessary (w,h) constructor.